build/tutorials/tutorials-contest/red-dev-avalanche-chainlink-tutorial/chainlink-external-adaptor/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Remove package.json and package-lock.json

Reply:
Removed the files. But, can you explain why don't we need the files?